.github/dependabot.yml

```diff
@@ -0,0 +1,15 @@
+# Please see the documentation for all configuration options:
+# https://help.github.com/github/administering-a-repository/configuration-options-for-dependency-updates
+
+version: 2
+updates:
+- package-ecosystem: "github-actions"
+  directory: "/"
```

Comment:
Can't / shouldn't we limit this to `/.github/workflows`?

Reply:
No, github-actions is a special case, I have added a link to the documentation.